### DISCUSSION TITLE
Unable to find logs sent from Cisco IOS XR device

### DISCUSSION THREAD

**Discussion by anpedro** [syslog-ng/syslog-ng#4442] [General]
Hello,
 I am sending syslog messages from IOS XR device to server where I am running syslog-ng inside container.

I can see the syslog messages hitting the interface in the linux server capturing via tcpdump, however, it doesn't look like syslog-ng detects those messages:
`
[root@logging-logstash:/var/containers/syslog-ng]# tcpdump -i ens224 port 514
dropped privs to tcpdump
tcpdump: verbose output suppressed, use -v[v]... for full protocol decode
listening on ens224, link-type EN10MB (Ethernet), snapshot length 262144 bytes
22:24:29.713427 IP6 2001:10:8:100::1.syslog > logging-logstash.syslog: SYSLOG local7.notice, length: 219
22:24:29.718957 IP6 2001:10:8:100::1.syslog > logging-logstash.syslog: SYSLOG local7.notice, length: 225
22:24:29.725462 IP6 2001:10:8:100::1.syslog > logging-logstash.syslog: SYSLOG local7.notice, length: 225
22:24:29.731921 IP6 2001:10:8:100::1.syslog > logging-logstash.syslog: SYSLOG local7.notice, length: 225
22:24:29.738424 IP6 2001:10:8:100::1.syslog > logging-logstash.syslog: SYSLOG local7.notice, length: 225
22:24:29.744833 IP6 2001:10:8:100::1.syslog > logging-logstash.syslog: SYSLOG local7.notice, length: 225
`

Can you please review my configuration below and make sure it's correct? the end goal of this exercise is to send these messages to promtail-loki and look at this data from grafana.

`
[root@logging-logstash:/var/containers/syslog-ng]# cat syslog-ng.conf
#@version: 3.5
@version: 4.0
@include "scl.conf"
source s_local {
    internal();
};
source s_network {
    udp(port("514"));
};
#source s_net {
#   udp(
#     ip("0.0.0.0")
#   );
#   syslog(
#     ip("0.0.0.0")
#   );
# };
destination d_loki {
    syslog("promtail" transport("tcp") port("1514"));
};
log {
#        source(s_net);
        source(s_local);
        source(s_network);
        destination(d_loki);
};
`

4240d758fa79  docker.io/balabit/syslog-ng:latest                        8 minutes ago  Up 8 minutes ago (healthy)  :::514->514/udp                                 containers_syslog-ng_1

[root@logging-logstash:/var/containers/syslog-ng]# podman logs -f containers_syslog-ng_1
syslog-ng: Error setting capabilities, capability management disabled; error='Operation not permitted'
[2023-04-25T02:18:52.337392] You have a TLS enabled source without a X.509 keypair. Make sure you have tls(key-file() and cert-file()) options, TLS handshake to this source will fail; location='/usr/share/syslog-ng/include/scl/default-network-drivers/plugin.conf:58:26' <-- I don't see any logs here.

**Comment by bazsi** [comment]
My guess would be that your packet filter filters these out or an ingress
is missing on that port. Your config seems to be ok.

**Reply by anpedro** [reply]
thanks Bazsi,

How do I figure this one out? I've brought up the container using -edv but I don't seem to be much from the logs. 
We are in rush to implement this solution and I'd appreciate any help provided.

**Comment by anpedro** [comment]
packet filter you mean something like SELinux or iptables?

**Comment by bazsi** [comment]
Can you log into our discord server? Would be easier to chat about that.

https://discord.gg/E65kP9aZGm

I would think of iptables.

You could try running the container in the host network namespace. That
should be the same as if running locally.
